ksicht/templates/includes/footer.html

```diff
@@ -5,7 +5,8 @@
                 <p>
                     <strong>KSICHT</strong> ~ <strong>K</strong>orespondenční <strong>S</strong>eminář <strong>I</strong>nspirovaný <strong>C</strong>hemickou <strong>T</strong>ematikou.
                 </p>
-                <p>Zdrojový kód je licencován pod <a href="https://www.gnu.org/licenses/agpl-3.0.html" target="_blank" rel="noopener">AGPL-3.0</a> a je dostupný na <a href="https://github.com/xaralis/ksicht" target="_blank" rel="noopener">GitHubu</a>.</p>
+                <p style="margin-bottom: 0; ">Zdrojový kód je licencován pod <a href="https://www.gnu.org/licenses/agpl-3.0.html" target="_blank" rel="noopener">AGPL-3.0</a> a je dostupný na <a href="https://github.com/xaralis/ksicht" target="_blank" rel="noopener">GitHubu</a>.</p>
```

Comment:
No inline styles please.

Reply:
replaced with `has-margin-b-0` and `has-margin-t-0`